### DISCUSSION TITLE
MCMC for lens_model_DESIJ1101_0602

### DISCUSSION THREAD

**Discussion by rafee-adnan** [AstroBridge/BDLensing#237] [Q&A]
As I suspected, when I lower the limit the gamma values drops with the limit. As you have said @ajshajib I have re-run the MCMC freeing gamma lowering it's limit to 1.5. Here are the trace plot and corner plot. Kindly suggest me what can I do in this situation.

![image](https://github.com/ajshajib/BDLensing/assets/76589056/6ee79465-d92c-4c53-9a8f-4900f3cafce8)
![image](https://github.com/ajshajib/BDLensing/assets/76589056/77d4e8b0-b3d1-4636-8cdd-822cd5eacdde)

**Comment by ajshajib** [comment]
You can try using a Gaussian prior for `gamma` with a mean of 2.09 and a standard deviation of 0.35 (based on this paper: https://www.projectdinos.com/dinos-i). In `lenstronomy`, you can provide it using:

```
kwargs_likelihood = {
    ...
    'prior_lens': [[0, "gamma", 2.09, 0.35]],
    ...
}
```

**Reply by rafee-adnan** [reply]
This is the latest mcmc run using `"prior_lens": [[0, "gamma", 2.09, 0.35]]`. It is still heading towards the same direction.

![image](https://github.com/ajshajib/BDLensing/assets/76589056/f5e75c20-0ead-46db-9350-12d004653ff3)

**Reply by ajshajib** [reply]
Ok, can you kindly check if 2.08±0.13 would help more, based on [this paper](https://ui.adsabs.harvard.edu/abs/2021MNRAS.503.2380S/abstract)?
